### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by GingerC75** [siemens/ix#2395]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

In the general application layout there is no banner landmark set which contains the bypass-links which are not present at all, it also contains the logo and the link to the application.

### What type of frontend framework are you seeing the problem on?

Angular (NgModule)

### On which version of the frontend framework are you experiencing the issue?

latest

### Which version of iX do you use?

v4

### Code to produce this issue.

```shell
no code to provide since it is missing in total. refer to W3C WAI WCAG2.2 success criteria 2.4.1 bypass blocks.
```
